presto-iceberg/src/main/java/com/facebook/presto/iceberg/IcebergSplitManager.java

```diff
@@ -13,6 +13,7 @@
  */
 package com.facebook.presto.iceberg;
 
+import com.facebook.presto.common.predicate.TupleDomain;
```

Comment:
The commit message is too long

Reply:
I modified the message

Reply:
Still too long. how about "Prune Iceberg partitions when filter pushdown is enabled"?